cli/dcoscli/package/main.py

```diff
@@ -334,12 +334,28 @@ def _install(package_name, package_version, options_path, app_id, cli,
     pkg = package_manager.get_package_version(package_name, package_version)
 
     pkg_json = pkg.package_json()
+
+    selected = pkg_json.get('selected')
+    if selected:
+        link = ('https://mesosphere.com/'
+                'catalog-terms-conditions/#certified-services')
+    else:
+        link = ('https://mesosphere.com/'
+                'catalog-terms-conditions/#community-services')
+    emitter.publish(
+        ('By Deploying, you agree to '
+         'the Terms and Conditions ' + link)
+    )
+
     pre_install_notes = pkg_json.get('preInstallNotes')
     if app and pre_install_notes:
         emitter.publish(pre_install_notes)
-        if not confirm('Continue installing?', yes):
-            emitter.publish('Exiting installation.')
-            return 0
+
+    if yes:
+        emitter.publish('Continue installing? [yes/no] ')
```

Comment:
I think I wasn't 100% accurate in my previous comment. I forgot about `--yes`. We should preserve the old behavior and not print this if the user passes `--yes`. In other words I think this should just be:
```python
if not confirm('Continue installing?', yes):
  emitter.publish('Exiting installation.')
  return 0
```